fastlane/Fastfile

```diff
@@ -549,17 +548,46 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
+    }.each { |source, destination|
+      source_dir = Dir.pwd + "/metadata/" + source
+      Dir.children(source_dir).select { |file| 
+        FileUtils.mv source_dir + "/" + file, Dir.pwd + "/metadata/" + destination + "/" + file
+      }
+      FileUtils.remove_dir(source_dir)
+    }
+
+    # Copy source files into `en-US`
+    enUS_path = Dir.pwd + "/metadata/en-US/"
+
+    FileUtils.mkdir_p(enUS_path) unless File.directory?(enUS_path)
 
-      support_path = Dir.pwd + "/metadata/#{locale}/support_url.txt"
-      write_file(support_path, "https://docs.woocommerce.com/")
+    [
+      "promo_screenshot_1.txt",
+      "promo_screenshot_2.txt",
+      "promo_screenshot_3.txt",
+      "promo_screenshot_4.txt",
+      "promo_screenshot_5.txt",
+    ].each { |source|
+      FileUtils.cp( Dir.pwd + "/appstoreres/metadata/source/" + source, enUS_path + source)
     }
+
+    # Replace the `default` directory with the contents of `en-US`
+    default_path = Dir.pwd + "/metadata/default/"
+    FileUtils.remove_dir(default_path) if File.directory?(default_path)
+    FileUtils.copy_entry(enUS_path, default_path)
+
+    # Never store review information (it contains passwords and PII)
+    review_info_path = Dir.pwd + "/metadata/review_information/"
```

Comment:
…Will you guess my suggestion for this one?… 😛 

```suggestion
    review_info_path = File.join(Dir.pwd, 'metadata', 'review_information')
```

Reply:
Fixed in 49ee666b68863303a8b29c33f6a4aba413e27109